src/pystatis/table.py

```diff
@@ -133,6 +134,14 @@ def parse_genesis_table(data: pd.DataFrame) -> pd.DataFrame:
         # Extracts time column with name from first element of Zeit_Label column
         time = pd.DataFrame({data["Zeit_Label"].iloc[0]: data["Zeit"]})
 
+        # Some tables of Genesis can have a regional code (AGS) as first attribute
+        ags_code = None
+        pos_of_ags_col = np.where(data.iloc[0].isin(config.REGIO_AGS_CODES))[0]
+        if pos_of_ags_col.size > 0:
+            pos_of_ags_col = pos_of_ags_col[0]
+            label = "Amtlicher Gemeindeschlüssel (AGS)"  # en: official municipality code (AGS)
+            ags_code = pd.Series(data=data.iloc[:, pos_of_ags_col + 2], name=label)
```

Comment:
A question: Usually we address the data by column names, which also seems more robust against database and formatting changes. Is this possible and desirable for extracting the AGS codes as well?

Reply:
No, otherwise I would have done that. You cannot do this here or only with much more effort like extracting the label and build the new one. The idea here is that I **know** that the second next column contains the code labels/values to the AGS code column, so I explicitly don't rely on labels but on format and structure which in my opinion is always better than relying on labels, just think on support de and en and having differnet labels